src/snowflake/snowpark/functions.py

```diff
@@ -3668,3 +3690,37 @@ def sproc(
             execute_as=execute_as,
             strict=strict,
         )
+
+
+# Add these alias for user code migration
+call_builtin = call_function
+builtin = function
+countDistinct = count_distinct
+substr = substring
+to_varchar = to_char
+expr = sql_expr
+date_format = to_date
+first = any_value
+create_map = object_construct
+map_keys = object_keys
```

Comment:
Snowflake SQL is creating a new data type Map. How about reserve create_may and map_keys for now?

Reply:
removed